### ISSUE TITLE
Using "write_phyloseq" command to convert a phyloseq object to a dataframe

### ISSUE THREAD

**Issue by enriqueza** [joey711/phyloseq#1355]
Hello everyone,

I am navigating through phyloseq and google, and thought I found a command to easily convert a phyloseq object to a dataframe so I would not have to deal with phyloseq anymore. I tried running "write_phyloseq" as a command I found, but Rstudio said it could not find the command. I  have the latest R version installed, so I was wondering if this is an actual command? If not, is there another command/set of commands that will do this? Any help is welcome. I've looked through the issues about this but haven't really found a solution to converting a whole phyloseq object to a dataframe which can be saved as a .csv file. Any help is welcome.

**Comment by mikemc** [comment]
This is not a phyloseq command. You can convert a phyloseq object to a data frame with `psmelt()`, and then write that with `write.csv()`, but my guess is that isn't what you want. Are you sure you don't want to write the otu table, sample data, tax table, etc as separate csv files? What do you want the csv files for?

**Comment by enriqueza** [comment]
I more or less solved it, it just took forever. I did use the `write.csv()` function by calling my phyloseq object "ps" and then ps@otu_table and ps@tax_table. After downloading them separately I had to open them back up, flip around the OTU table rows and columns because it was really huge and took forever to open, and then merged it with the taxa table. Essentially I wanted to be able to plot the OTUs using vegan or some other package other than phyloseq.
